supriya/nonrealtime/renderer.py

```diff
@@ -114,9 +114,9 @@ def _build_render_command(
         cwd = pathlib.Path.cwd()
         scsynth_path = scsynth.find(scsynth_path)
         command = [scsynth_path]
-        server_options_serial = (server_options or scsynth.Options()).serialize(
-            realtime=False
-        )
+        server_options_serial = (
+            server_options or scsynth.Options(realtime=False)
+        ).serialize()
```

Comment:
Since `realtime` is becoming a part of `Options`, we need to guard against the possibility of someone passing an options instance with `realtime=True` here.

Something like `server_options_serial = new(server_options or scsynth.Options(), realtime=False).serialize()` would probably do it: use the existing options or a new one, and override the realtime setting regardless.